nginx/assets/configuration/spec.yaml

```
@@ -71,6 +71,11 @@ files:
       overrides:
         disable_generic_tags.hidden: False
         disable_generic_tags.enabled: True
+        disable_generic_tags.description: |
+          Generic tags such as `cluster` will be replaced by <integration_name>_cluster to avoid
+          getting mixed with other integration tags.
+          Note that this only relevant to service check submissions, metrics do not include any generic tags to begin with.
```

```suggestion
          Note that this only relevant to service check submissions, metrics do not include any generic tags.
```

nit- I think it's not needed